.gitignore

```diff
@@ -6,10 +6,15 @@ coverage
 .nyc_output
 package-lock.json
 packages/amplify-cli/amplify/.config/
+packages/amplify-storage-simulator/public
 .vscode
 packages/graphql*/lib
+packages/amplify-storage-simulator/lib
+packages/amplify-storage-simulator/node_modules
 packages/amplify-graphql*/lib
 packages/amplify-e2e-tests/lib
 packages/*/node_modules
 packages/*/package-lock.json
+packages/amplify-util-mock/lib
+packages/amplify-util-mock/node_modules
```

Comment:
This is not needed as we have a general rule on line#16

Reply:
done